File: 프로그래머스/lv0/120845. 주사위의 개수/주사위의 개수.java
class Solution {
    public int solution(int[] box, int n) {
        int answer = 1;
        // answer에 각 모서리별 값을 곱해야 하므로 1을 넣어준다		
		for(int i = 0; i < box.length; i++) {
			answer *= (box[i] / n);
		} // box의 모서리별 n크키의 상자가 들어가는 수만큼 곱해준다
        return answer;
    }
}

File: 프로그래머스/0/120845. 주사위의 개수/주사위의 개수.js
function solution(box, n) {
    let answer = 1;
    // answer에 각 모서리별 값을 곱해야 하므로 1을 넣어준다		
    for (let i = 0; i < box.length; i++) {
        answer *= Math.floor(box[i] / n);
    } // box의 모서리별 n크키의 상자가 들어가는 수만큼 곱해준다
    return answer;
}
